kebab.json

```diff
@@ -1,28 +1,28 @@
 {
 	"response": [
 		{
-			"maintainer": "",
+			"maintainer": "Bojan Tanasic (bojantanasic)",
 			"oem": "OnePlus",
 			"device": "OnePlus 8T",
-			"filename": "crDroidAndroid-14.0-20240412-kebab-v10.4.zip",
-			"download": "https://sourceforge.net/projects/crdroid/files/kebab/10.x/crDroidAndroid-14.0-20240412-kebab-v10.4.zip/download",
-			"timestamp": 1712923869,
-			"md5": "d3d576b4042ab1cf76572a2477008e38",
-			"sha256": "2fdd83902297306080cc41f1714e91754c674c2c09bd892a0d112010803af0be",
-			"size": 1611827241,
-			"version": "10.4",
+			"filename": "crDroidAndroid-14.0-20240920-kebab-v10.8.zip",
+			"download": "https://sourceforge.net/projects/crdroid10/files/crDroidAndroid-14.0-20240920-kebab-v10.8.zip/download",
+			"timestamp": 1726862857,
+			"md5": "dd9f6efde09f30f27ad44d437955acf0",
+			"sha256": "a021c397fb9bf888da38ecb1bc21b8c431a0f1466e4a3e16723aa9e92a7877e4",
+			"size": 1499343089,
+			"version": "10.8",
 			"buildtype": "Monthly",
-			"forum": "https://forum.xda-developers.com/t/rom-13-op8-8-pro-8t-crdroid-v9-0-23-12-2022.4533151/",
+			"forum": "https://xdaforums.com/t/rom-14-official-8t-crdroid-v10-8-14-09-2024.4692700/",
 			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-U/",
 			"firmware": "",
 			"modem": "",
 			"bootloader": "",
-			"recovery": "https://sourceforge.net/projects/crdroid/files/kebab/10.x/recovery/",
-			"paypal": "https://paypal.me/saurabhparab007",
-			"telegram": "https://t.me/crdroidop8",
-			"dt": "https://github.com/crdroidandroid/android_device_oneplus_kebab",
-			"common-dt": "https://github.com/crdroidandroid/android_device_oneplus_sm8250-common",
-			"kernel": "https://github.com/crdroidandroid/android_kernel_oneplus_sm8250"
+			"recovery": "https://sourceforge.net/projects/crdroid10/files/",
+			"paypal": "https://www.paypal.com/paypalme/bojantanasic",
+			"telegram": "https://t.me/crDroid10",
+			"dt": "https://github.com/bojantanasic/device_oneplus_kebab",
```

Comment:
dt needs to be hosted on crdroid git
entries were ok, but source was not pushed

Reply:
now it`s pushed on a branch 14

Reply:
This Branches looks horrible 
Please clean it
Also why not push it to 14-qpr3.